sim/Makefile

```diff
@@ -60,6 +65,8 @@ SBT ?= java $(JAVA_OPTS) -jar $(rocketchip_dir)/sbt-launch.jar $(SBT_OPTS) $(SBT
 define run_scala_main
 	cd $(base_dir) && $(SBT) ";project $(1); runMain $(2) $(3)"
 endef
+# recipes for generator and gg using sbt are calls out to FAT_JAR instead of SBT
+run_main = cd $(base_dir) && java $(JAVA_ARGS) -cp $(FAT_JAR) $(2) $(3)
```

Comment:
Reminder to change all `JAVA_ARGS` to `JAVA_OPTS` to match the change done in Chipyard.

Reply:
addressed in 7c299da of the rebase I just force-pushed.